plugins/plugin-proxy-support/src/lib/proxy-executor.ts

```diff
@@ -33,7 +33,21 @@ const debug = Debug('plugins/proxy-support/executor')
  *
  */
 import defaultProxyServerConfig = require('@kui-shell/proxy/lib/defaultProxyServerConfig.json')
-const proxyServerConfig = config['proxyServer'] || defaultProxyServerConfig
+
+// TODO: we must have some way to inject traiv's config nicely, maybe build a custom client???
```

Comment:
in clients/default/theme/config.json, one can inject values; the contents of that file will be available as

```
import { config } from '@kui-shell/core/core/settings'
```